test/kafka_test.go

```diff
@@ -81,7 +83,7 @@ func (s *CLITestSuite) TestKafka() {
 		{args: "kafka cluster describe lkc-describe-infinite -o yaml", fixture: "kafka/43.golden"},
 
 		{args: "kafka cluster describe lkc-describe-dedicated-provisioning", fixture: "kafka/cluster-describe-dedicated-provisioning.golden"},
-		{args: "kafka cluster describe lkc-describe-topic-error", fixture: "kafka/cluster-describe-no-topic-count.golden"},
+		{args: "kafka cluster describe lkc-describe-topic-error -vv", contains: errors.OmitTopicCountMsg},
```

Comment:
Not a huge fan of `contains`... can we just print the complete output?

Reply:
I switched to `contains` because `-vv` introduces a timestamp into the output, so I can't easily put that in a golden file, which is annoying.

Reply:
Oh, I didn't see the `-vv`. In general, testing the verbose output doesn't seem super necessary so we've never done it.